testData/solutions/win32.json5

```diff
@@ -6150,8 +6143,19 @@
                         "schema": {
                             "title": "Touchpad sensitivity",
                             "description": "Changes Windows 10 precision touchpad sensitivity.",
-                            "type": "string",
-                            "default": "medium"
+                            "default": "Medium sensitivity",
```

Comment:
Why did the default change?   I assume you were transforming to the actual value of `Medium sensitivity` previously?

Reply:
You are correct, previously I was transforming it.